content/en/docs/community/release_checklist.md

```diff
@@ -4,44 +4,67 @@ description: "Checklist for maintainers when releasing the next version of Helm.
 weight: 2
 ---
 
-**IMPORTANT**: If your experience deviates from this document, please document
-the changes to keep it up-to-date.
+# A Maintainer's Guide to Releasing Helm
 
-## A Maintainer's Guide to Releasing Helm
-
-So you're in charge of a new release for helm? Cool. Here's what to do...
-
-![TODO: Nothing](../../images/nothing.png)
-
-Just kidding! :trollface:
+Time for a new Helm release! As a Helm maintainer cutting a release, you are
+the best person to [update this
+release checklist](https://github.com/helm/helm-www/blob/master/content/en/docs/community/release_checklist.md)
+should your experiences vary from what's documented here.
 
 All releases will be of the form vX.Y.Z where X is the major version number, Y
 is the minor version number and Z is the patch release number. This project
 strictly follows [semantic versioning](https://semver.org/) so following this
 step is critical.
 
+These directions will cover initial configuration followed by the release
+process for three different kinds of releases:
+
+* Major Releases - released less frequently - may have breaking changes
+* Minor Releases - released regularly - no breaking changes
+* Patch Releases - released as needed - do not require all steps in this guide
```

Comment:
I experimented with separating this into three guides but ultimately decided that one guide is okay, with a little bit more info about how to parse it.